<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>1024</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="selectImageBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #fff;
background-color: #d35400;
border-color: #357ebd;
font-size: 14px;
font-weight: normal;
border-radius: 4px;
text-align: center;
vertical-align: middle;
border: 1px solid transparent;
padding: 10px 16px;</string>
    </property>
    <property name="text">
     <string>Select image</string>
    </property>
   </widget>
   <widget class="QLabel" name="firstImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>391</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="secondImage">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>120</y>
      <width>391</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="mergedImages">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>120</y>
      <width>391</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="selectImageBtn2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #fff;
background-color: #d35400;
border-color: #357ebd;
font-size: 14px;
font-weight: normal;
border-radius: 4px;
text-align: center;
vertical-align: middle;
border: 1px solid transparent;
padding: 10px 16px;</string>
    </property>
    <property name="text">
     <string>Select image</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>70</y>
      <width>391</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>22</width>
      <height>531</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="mergeImagesBtn">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">:enabled {
color: #fff;
background-color: #428bca;
border-color: #357ebd;
font-size: 14px;
font-weight: normal;
border-radius: 4px;
text-align: center;
vertical-align: middle;
border: 1px solid transparent;
padding: 10px 16px;
}

:disabled {
color: #fff;
background-color: #dddddd;
border-color: #357ebd;
font-size: 14px;
font-weight: normal;
border-radius: 4px;
text-align: center;
vertical-align: middle;
border: 1px solid transparent;
padding: 10px 16px;
}</string>
    </property>
    <property name="text">
     <string>Merge images</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveImageBtn">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">:enabled {
color: #fff;
background-color: #2ecc71;
border-color: #357ebd;
font-size: 14px;
font-weight: normal;
border-radius: 4px;
text-align: center;
vertical-align: middle;
border: 1px solid transparent;
padding: 10px 16px;
}
:disabled {
color: #fff;
background-color: #dddddd;
border-color: #357ebd;
font-size: 14px;
font-weight: normal;
border-radius: 4px;
text-align: center;
vertical-align: middle;
border: 1px solid transparent;
padding: 10px 16px;
}</string>
    </property>
    <property name="text">
     <string>Save image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>660</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>100</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>verticalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_4</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>563</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>572</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_5</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>449</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
